aviary/mission/flight_phase_builder.py

```diff
@@ -236,7 +237,7 @@ def build_phase(self, aviary_options: AviaryValues = None, phase_type='height_en
 
         phase.add_timeseries_output(Dynamic.Mission.ALTITUDE)
 
-        if phase_type == 'two_dof':
+        if phase_type is EquationsOfMotion.SOLVED_2DOF:
```

Comment:
I think this is supposed to be regular 2DOF?

Reply:
Nah, this is all using the solved two-dof setup for robustness